rust/processor/migrations/2023-10-20-212109_create_cockroach_tables/up.sql

```diff
@@ -0,0 +1,88 @@
+-- Your SQL goes here
+
+-- transactions_cockroach
+CREATE TABLE IF NOT EXISTS transactions_cockroach (
+  transaction_version BIGINT UNIQUE PRIMARY KEY NOT NULL,
+  transaction_block_height BIGINT NOT NULL,
+  hash VARCHAR(66) NOT NULL,
+  transaction_type VARCHAR(50) NOT NULL,
+  payload jsonb,
+  state_change_hash VARCHAR(66) NOT NULL,
+  event_root_hash VARCHAR(66) NOT NULL,
+  state_checkpoint_hash VARCHAR(66),
+  gas_used NUMERIC NOT NULL,
+  success BOOLEAN NOT NULL,
+  vm_status TEXT NOT NULL,
+  accumulator_root_hash VARCHAR(66) NOT NULL,
+  num_events BIGINT NOT NULL,
+  num_write_set_changes BIGINT NOT NULL,
+  epoch BIGINT NOT NULL,
+  parent_signature_type TEXT,
+  sender VARCHAR(66),
+  sequence_number BIGINT,
+  max_gas_amount NUMERIC,
+  expiration_timestamp_secs TIMESTAMP,
+  gas_unit_price NUMERIC,
+  "timestamp" TIMESTAMP,
+  entry_function_id_str TEXT,
+  signature jsonb,
+  id VARCHAR(66),
+  round BIGINT,
+  previous_block_votes_bitvec jsonb,
+  proposer VARCHAR(66),
+  failed_proposer_indices jsonb,
+  inserted_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+CREATE INDEX txn_cockroach_insat_index ON transactions_cockroach (inserted_at);
```

Comment:
do we even need this? we had to add this b/c we couldn't make replication work.

Reply:
ah, i guess we dont need it